android/src/main/java/com/phoebe/pbwebview/PBWebViewManager.java

```diff
@@ -533,7 +533,10 @@ public void onPageStarted(WebView view, String url, Bitmap favicon) {
             event.putBoolean("canGoBack", webView.canGoBack());
             event.putBoolean("canGoForward", webView.canGoForward());
             dispatchEvent(webView, PBWebViewEvent.createNewWindowEvent(webView.getId(), event));
-            webView.removeView(newView);
+            try {
+              webView.removeView(newView);
+              newView.destroy();
+            } catch (Exception e) {}
```

Comment:
I think we need to write something into exception handler

Reply:
The part is just to make sure `newView` is removed. Exception if occurs here only mean that the `newView` was removed before. That's why I did not do anything.

Reply:
Let me put the explanation in exception.